koku/masu/database/trino_sql/openshift/cost_model/monthly_cost_gpu.sql

```diff
@@ -0,0 +1,107 @@
+INSERT INTO postgres.{{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary (
+    uuid,
+    report_period_id,
+    cluster_id,
+    cluster_alias,
+    data_source,
+    usage_start,
+    usage_end,
+    namespace,
+    node,
+    resource_id,
+    pod_labels,
+    all_labels,
+    source_uuid,
+    cost_model_rate_type,
+    cost_model_cpu_cost,
+    cost_model_memory_cost,
+    cost_model_volume_cost,
+    monthly_cost_type,
+    cost_category_id
+)
+SELECT
+    cast(uuid() as varchar) as uuid,
+    {{report_period_id}} as report_period_id,
+    {{cluster_id}} as cluster_id,
+    {{cluster_alias}} as cluster_alias,
+    'GPU' as data_source,
+    date(gpu.interval_start) as usage_start,
+    date(gpu.interval_start) as usage_end,
+    gpu.namespace,
+    gpu.node,
+    gpu.gpu_uuid as resource_id,
+    json_format(cast(map(
+        ARRAY['gpu-model', 'gpu-vendor', 'gpu-memory-mib', 'pod-name'],
+        ARRAY[
+            gpu.gpu_model_name,
+            gpu.gpu_vendor_name,
+            CAST(gpu.gpu_memory_capacity_mib AS varchar),
+            gpu.pod
+        ]
+    ) as json)) as pod_labels,
+    json_format(cast(map(
+        ARRAY['gpu-model', 'gpu-vendor', 'gpu-memory-mib', 'pod-name'],
+        ARRAY[
+            gpu.gpu_model_name,
+            gpu.gpu_vendor_name,
+            CAST(gpu.gpu_memory_capacity_mib AS varchar),
+            gpu.pod
+        ]
+    ) as json)) as all_labels,
+    CAST(gpu.source AS varchar) as source_uuid,
+    {{rate_type}} AS cost_model_rate_type,
+    -- GPU cost calculation: (rate / days_in_month) * (uptime_seconds / 86400)
+    -- Formula: daily_rate * uptime_as_fraction_of_day
+    {%- if rate is defined %}
+    (CAST({{rate}} AS decimal(24,9)) / CAST({{amortized_denominator}} AS decimal(24,9))) * (gpu.gpu_pod_uptime / 86400.0),
+    {%- elif value_rates is defined %}
+    CASE
+        {%- for value, value_rate in value_rates.items() %}
+        {%- if value.startswith('{') %}
+        {%- set rate_spec = value | from_json %}
+        -- JSON format: {"model": "A100", "vendor": "NVIDIA"}
```

Comment:
This is not the correct json format:
```
'tag_key': 'gpu_model', 'default_rate': 5000.0, 'value_rates': {'Tesla T4': 1000.0, 'A100': 2500.0}
```

The tag key here is the gpu model, then the value_rates contains the vendor & rate.